index.html

```diff
@@ -295,10 +295,10 @@ <h3>Expected Behavior</h3>
     <h2>Reporting</h2>
     <p>If you are concerned about your immediate safety, contact 
       <a href="https://en.wikipedia.org/wiki/List_of_emergency_telephone_numbers">local law enforcement</a>.
-      For a face to face event Chairs and venue staff should provide any necessary assistance contacting them.
+      For a face to face event Chairs and venue staff should provide any necessary assistance contacting law enforcement.
     </p>
     
-    <p>You may also contact a W3C ombud who will assist you. Contact addresses are included in the
+    <p>You may also contact a W3C ombuds who will assist you. Contact addresses are included in the
```

Comment:
Having looked again today, I think we should use "ombudsperson" rather than just "ombuds".